alf/algorithms/trac_algorithm.py

```diff
@@ -61,9 +63,6 @@ class TracAlgorithm(ActorCriticAlgorithm):
 
     def __init__(self,
                  action_spec,
-                 observation_spec,
```

Comment:
It seems that other parameters can also be removed

Reply:
ActorCriticAlgorithm needs other parameters.

Reply:
i see